apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1738,33 +1761,26 @@ dep_tx_for_signing(Opts, D) ->
 wdraw_tx_for_signing(Opts, D) ->
     new_onchain_tx_for_signing(channel_withdraw_tx, Opts, D).
 
-close_mutual_tx_for_signing(LatestSignedTx, D) ->
+close_mutual_tx_for_signing(D) ->
     Account = my_account(D),
-    {ok, Nonce} = aec_next_nonce:pick_for_account(Account),
-    close_mutual_tx_(Account, Nonce, LatestSignedTx, D).
+    new_close_mutual_tx(#{ acct => Account }, D).
 
-close_mutual_tx_(Account, Nonce, _LatestSignedTx,
-                 #data{ on_chain_id = ChanId
-                      , opts        = Opts
-                      , state       = State} = D) ->
-    Def = close_mutual_defaults(Account, D),
-    Opts1 = maps:merge(Def, Opts),
-    #{initiator := Initiator,
-      responder := Responder} = Opts,
-    {ok, IAmt} = aesc_offchain_state:balance(Initiator, State),
-    {ok, RAmt} = aesc_offchain_state:balance(Responder, State),
-    Fee = maps:get(fee, Opts1),
-    TTL = maps:get(ttl, Opts1, 0), %% 0 means no TTL limit
-    {IAmt1, RAmt1} = pay_close_mutual_fee(Fee, IAmt, RAmt),
-    {LastRound, _} = aesc_offchain_state:get_latest_signed_tx(State),
-    aesc_close_mutual_tx:new(#{ channel_id             => aec_id:create(channel, ChanId)
-                              , from_id                => aec_id:create(account, Account)
-                              , initiator_amount_final => IAmt1
-                              , responder_amount_final => RAmt1
-                              , ttl                    => TTL
-                              , fee                    => Fee
-                              , round                  => LastRound + 1
-                              , nonce                  => Nonce }).
+fake_close_mutual_tx(RealCloseTx, D) ->
```

Comment:
A comment in the code to explain what a fake and a new close are could be valuable

Reply:
@ThomasArts Added a comment in the latest commit.